Test failure in ydb/tests/compatibility/test_fulltext_index.py.TestFulltextIndex.test_fulltext_index[rolling_prestable-26-1_to_prestable-26-2]

ydb/tests/compatibility/test_fulltext_index.py.TestFulltextIndex.test_fulltext_index[rolling_prestable-26-1_to_prestable-26-2]

**Test Failure**

**Build Information:**
- Version: 5b365c8c66bb6df0b1f822d21a5fa92f5943e9b2
- Build Preset: relwithdebinfo
- Branch: main

**Test Report:** [View report](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/21834015376/ya-main-x86-64/try_1/ya-test.html#FAIL)

**Run Command:**
```
./ya make -ttt --build "relwithdebinfo" -F 'test_fulltext_index.py::TestFulltextIndex::test_fulltext_index[rolling_prestable-26-1_to_prestable-26-2]' ydb/tests/compatibility
```

**Owner:** [TEAM:@ydb-platform/engineering](https://github.com/orgs/ydb-platform/teams/engineering)

**Test run history:** [link](https://datalens.yandex/34xnbsom67hcq?branch=main&full_name=ydb/tests/compatibility/test_fulltext_index.py.TestFulltextIndex.test_fulltext_index[rolling_prestable-26-1_to_prestable-26-2]&build_type=relwithdebinfo)

More info in [dashboard](https://datalens.yandex/4un3zdm0zcnyr)

**Logs:**
- [log](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/21834015376/ya-main-x86-64/try_1/artifacts/logs/ydb/tests/compatibility/test-results/py3test/testing_out_stuff/test_fulltext_index.py.TestFulltextIndex.test_fulltext_index.rolling_prestable-26-1_to_prestable-26-2.log)
- [logsdir](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/21834015376/ya-main-x86-64/try_1/test_artifacts/ydb-tests-compatibility.zip)

**Error Summary:**
```
ydb/tests/compatibility/test_fulltext_index.py:179: in test_fulltext_index
    self.wait_index_ready()
ydb/tests/compatibility/test_fulltext_index.py:81: in wait_index_ready
    assert wait_for(predicate, timeout_seconds=100, step_seconds=1), "Error getting index status"
ydb/tests/library/common/wait_for.py:16: in wait_for
    if predicate():
ydb/tests/compatibility/test_fulltext_index.py:74: in predicate
    self.select_from_index_without_roll()
ydb/tests/compatibility/test_fulltext_index.py:163: in select_from_index_without_roll
    self._do_queries(queries)
ydb/tests/compatibility/test_fulltext_index.py:149: in _do_queries
    result_sets = session_pool.execute_with_retries(query)
contrib/python/ydb/py3/ydb/query/pool.py:259: in execute_with_retries
    return retry_operation_sync(wrapped_callee, retry_settings)
contrib/python/ydb/py3/ydb/retries.py:163: in retry_operation_sync
    for next_opt in opt_generator:
contrib/python/ydb/py3/ydb/retries.py:118: in retry_operation_impl
    result = YdbRetryOperationFinalResult(callee(*args, **kwargs))
contrib/python/ydb/py3/ydb/query/pool.py:257: in wrapped_callee
    return [result_set for result_set in it]
contrib/python/ydb/py3/ydb/_utilities.py:176: in __next__
    return self._next()
contrib/python/ydb/py3/ydb/query/base.py:89: in _next
    raise e
contrib/python/ydb/py3/ydb/query/base.py:85: in _next
    return super()._next()
contrib/python/ydb/py3/ydb/_utilities.py:167: in _next
    res = self.wrapper(next(self.it))
contrib/python/ydb/py3/ydb/query/session.py:454: in <lambda>
    lambda resp: base.wrap_execute_query_response(
contrib/python/ydb/py3/ydb/query/base.py:191: in decorator
    return func(rpc_state, response_pb, session, *args, **kwargs)
contrib/python/ydb/py3/ydb/query/base.py:208: in wrap_execute_query_response
    issues._process_response(response_pb)
contrib/python/ydb/py3/ydb/issues.py:266: in _process_response
    raise exc_class(_format_response(response_proto), response_proto.issues)
E   ydb.issues.GenericError: message: "Type annotation" issue_code: 1030 severity: 1 issues { position { row: 2 column: 17 } message: "At function: RemovePrefixMembers" end_position { row: 2 column: 17 } severity: 1 issues { position { row: 6 column: 17 } message: "At function: Take" end_position { row: 6 column: 17 } severity: 1 issues { position { row: 2 column: 17 } message: "At function: PersistableRepr, At function: SqlProject" end_position { row: 2 column: 17 } severity: 1 issues { position { row: 4 column: 22 } message: "At function: Filter, At lambda, At function: Coalesce" end_position { row: 4 column: 22 } severity: 1 issues { position { row: 5 column: 33 } message: "At function: Apply, At function: Udf, At FullText.Contains" end_position { row: 5 column: 33 } severity: 1 issues { position { row: 5 column: 33 } message: "Failed to find UDF function: FullText.Contains, reason: Error: Module FullText is not registered" end_position { row: 5 column: 33 } severity: 1 } } } } } } (server_code: 400080)
```

--- a/ydb/tests/compatibility/test_fulltext_index.py
+++ b/ydb/tests/compatibility/test_fulltext_index.py
@@ -92,9 +92,6 @@ class TestFulltextIndex(RollingUpgradeAndDowngradeFixture):
         table_name = f"table_{text_type}"
         index_name = f"idx_{index_type}_{tokenizer}"
         queries = []
-        utf8 = ''
-        if text_type == 'utf8':
-            utf8 = 'Utf8'
         for i in range(self.query_count):
             query = ' '.join(self.good_queries[table_name][i])
             queries.append([
@@ -102,14 +99,14 @@ class TestFulltextIndex(RollingUpgradeAndDowngradeFixture):
                 SELECT `pk`, `text`
                 FROM `{table_name}`
                 VIEW `{index_name}`
-                WHERE FullText::Contains{utf8}(`text`, "{query}")
+                WHERE FulltextMatch(`text`, "{query}")
                 LIMIT {self.limit};
                 """
             ])
             if index_type == 'fulltext_relevance':
                 queries.append([
                     True, f"""
-                    SELECT `pk`, `text`, FullText::Relevance{utf8}(`text`, "{query}") as `rel`
+                    SELECT `pk`, `text`, FulltextScore(`text`, "{query}") as `rel`
                     FROM `{table_name}`
                     VIEW `{index_name}`
                     ORDER BY `rel`

--- a/ydb/tests/stress/oltp_workload/workload/type/fulltext_index.py
+++ b/ydb/tests/stress/oltp_workload/workload/type/fulltext_index.py
@@ -91,17 +91,13 @@ class WorkloadFulltextIndex(WorkloadBase):
         """
         self.client.query(delete_sql, False)
 
-    def _select_contains(self, index_name, table_path, utf8=False):
-        if utf8:
-            utf8 = "Utf8"
-        else:
-            utf8 = ""
+    def _select_contains(self, index_name, table_path):
         query = ' '.join(fulltext.get_random_words(3))
         select_sql = f"""
             SELECT `pk`, `text`
             FROM `{table_path}`
             VIEW `{index_name}`
-            WHERE FullText::Contains{utf8}(`text`, "{query}")
+            WHERE FulltextMatch(`text`, "{query}")
             LIMIT {self.limit};
         """
         res = self.client.query(select_sql, False)
@@ -111,14 +107,10 @@ class WorkloadFulltextIndex(WorkloadBase):
         logger.info(f"Selected {n} rows using contains")
         return n
 
-    def _select_relevance(self, index_name, table_path, utf8=False):
-        if utf8:
-            utf8 = "Utf8"
-        else:
-            utf8 = ""
+    def _select_relevance(self, index_name, table_path):
         query = ' '.join(fulltext.get_random_words(3))
         select_sql = f"""
-            SELECT `pk`, `text`, FullText::Relevance{utf8}(`text`, "{query}") as `rel`
+            SELECT `pk`, `text`, FulltextScore(`text`, "{query}") as `rel`
             FROM `{table_path}`
             VIEW `{index_name}`
             ORDER BY `rel`
@@ -176,22 +168,20 @@ class WorkloadFulltextIndex(WorkloadBase):
         )
         n = 0
         for i in range(0, self.query_count):
-            # select from index with Fulltext::Contains
+            # select from index with FulltextMatch
             n += self._select_contains(
                 index_name=index_name,
                 table_path=table_path,
-                utf8=utf8,
             )
         if n == 0:
             raise Exception(f"No rows selected with {self.query_count} contains queries")
         if index_type == 'fulltext_relevance':
             n = 0
             for i in range(0, self.query_count):
-                # select from index with Fulltext::Relevance
+                # select from index with FulltextScore
                 n += self._select_relevance(
                     index_name=index_name,
                     table_path=table_path,
-                    utf8=utf8,
                 )
             if n == 0:
                 raise Exception(f"No rows selected with {self.query_count} relevance queries")

